app.rb

```diff
@@ -145,6 +149,28 @@ def config
 
     private
 
+    def get_ial_select_options(is_eipp_allowed:)
+      enhanced_ipp_option = ['enhanced-ipp-required', 'Enhanced In-Person Proofing (Enabled in dev & staging only)']
+      option = [
+        ['1', 'Authentication only (default)'],
+        ['2', 'Identity-verified'],
+        ['0', 'IALMax'],
+        ['step-up', 'Step-up Flow'],
+        ['biometric-comparison-required', 'Biometric Comparison (Disabled in prod)'],
+      ]
+
+      if is_eipp_allowed
+        option.push(enhanced_ipp_option)
+      else
+        option
+      end      
+    end 
```

Comment:
1. since this is all in the same class, this method has access to the helper method for eipp so we could probably remove the param entirely (and I'd also inline the options just to keep things scoped
2. I'd pluralize to `options` since it's an array

```suggestion
    def ial_select_options
      option = [
        ['1', 'Authentication only (default)'],
        ['2', 'Identity-verified'],
        ['0', 'IALMax'],
        ['step-up', 'Step-up Flow'],
        ['biometric-comparison-required', 'Biometric Comparison (Disabled in prod)'],
      ]

      if eipp_allowed?
        options << ['enhanced-ipp-required', 'Enhanced In-Person Proofing (Enabled in dev & staging only)']
      else

      options
    end 
```

Reply:
Resolved with commit ae25babbbada2aa7084ea3143a788d20e7e7e9c0